spec/indent/indent_spec.rb

```diff
@@ -76,6 +76,14 @@
     end
   end
 
+  describe "when after an '[' that is inside a byte string" do
+    before { vim.feedkeys 'itest = b"["\<CR>' }
+
+    it "does not indent" do
+      indent.should == 0
+    end
```

Comment:
This test does not make much sense, does it?  At least it is not failing on master, too.
For multiline strings `"""` should have been used (if the `<CR>` is meant to have happened after `[`).